javascript-if/main.js

```diff
@@ -0,0 +1,102 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+var shia = {
+  name: "Shia Lebouf",
+  age: 34,
+};
+
+function isOldEnoughToDrink(age) {
+  if ((age) => 21) {
```

Comment:
You have the characters backwards here. With "less than or equal to" and "greater than or equal to", the angle bracket comes first. with the equal sign first, you are using an entirely different operator we haven't talked about yet. This should be `>=`. You have this off in a couple other definitions too, double check your others while you fix this. Also, make sure you test your functions again to see if they are working as expected still